mbedtls-sys/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "mbedtls-sys-auto"
-version = "2.28.7"
+version = "2.28.8"
```

```suggestion
version = "2.28.7"
```
This is good. But it's better to put bump version into separate PR.